tsconfig.base.json

```diff
@@ -15,8 +15,6 @@
 		"noUnusedParameters": true,
 		"esModuleInterop": false,
 		"forceConsistentCasingInFileNames": true,
-		"ignoreDeprecations": "5.0",
-		"importsNotUsedAsValues": "error",
```

Comment:
Not strictly related but I believe this is in line with changes done in other repos - https://github.com/volarjs/volar.js/commit/65b1c0a2ea22ddf165b84db49ed9dddd32e6a377